packages/block-library/src/comment-template/edit.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * WordPress dependencies
+ */
+import { useState, useMemo } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import {
+	BlockContextProvider,
+	BlockPreview,
+	useBlockProps,
+	__experimentalUseInnerBlocksProps as useInnerBlocksProps,
+	store as blockEditorStore,
+} from '@wordpress/block-editor';
+import { Spinner } from '@wordpress/components';
+import { store as coreStore } from '@wordpress/core-data';
+
+const TEMPLATE = [
+	[ 'core/post-comment-author' ],
+	[ 'core/post-comment-date' ],
+	[ 'core/post-comment-content' ],
+];
+
+export default function CommentTemplateEdit( {
+	clientId,
+	context: { postId, queryPerPage },
+} ) {
+	const innerBlocksProps = useInnerBlocksProps( {}, { template: TEMPLATE } );
+	const blockProps = useBlockProps();
+
+	const [ activeBlockContext, setActiveBlockContext ] = useState();
+
+	const { comments, blocks } = useSelect(
+		( select ) => {
+			const { getEntityRecords } = select( coreStore );
+			const { getBlocks } = select( blockEditorStore );
+
+			return {
+				comments: getEntityRecords( 'root', 'comment', {
+					post: postId,
+					per_page: queryPerPage,
```

Comment:
Based on https://developer.wordpress.org/rest-api/reference/comments/#arguments those are default values, but it's better to be explicit here:

```suggestion
					status: 'approve',
					per_page: queryPerPage,
					order: 'asc',
```